### DISCUSSION TITLE
Can not connect the broker.emqx.io

### DISCUSSION THREAD

**Discussion by littletiger123** [lf-edge/ekuiper#1076] [Q&A]
**Please describe your problem in detail, if necessary, you can upload the log file through the attachment**:
I am a new one to use ekuiper and mosquitto. 
When I follow the quick_start_docker.md , I use the command
```
mosquitto_pub -h broker.emqx.io -m '{"temperature": 40, "humidity" : 20}' -t devices/device_001/messages
```
It returns 
```
Unable to connect (Lookup error.).
```
Thanks.
Also, I have a question that due to the  broker.emqx.io is the public broker, and if I will receive another publication from others whose topic is the same as mine?

**Comment by ngjaying** [comment]
> It returns
> 
> ```
> Unable to connect (Lookup error.).
> ```

This is likely a problem of your network environment, more specific DNS problem.

> Thanks. Also, I have a question that due to the broker.emqx.io is the public broker, and if I will receive another publication from others whose topic is the same as mine?

Yes, it is public. However, this is just for demo. You can easily switch to your private broker by setting `-e MQTT_SOURCE__DEFAULT__SERVERS=[tcp://yourbroker:1883]` in step 2.
